pkgs/by-name/db/db-rest/package.nix

```diff
@@ -2,15 +2,15 @@
   lib,
   buildNpmPackage,
   fetchFromGitHub,
-  nodejs_18,
+  nodejs_20,
   nix-update-script,
   nixosTests,
 }:
 buildNpmPackage rec {
   pname = "db-rest";
   version = "6.1.0";
 
-  nodejs = nodejs_18;
+  nodejs = nodejs_20;
```

Comment:
As the maintainer of the package, I originally pinned this to nodejs_18 since thats what upstream used, but we should unpin this so it always uses the default node version.

Reply:
I can do this change myself, or you can do it in this PR, whatever you prefer :)

Reply:
Either is fine by me, I've updated this PR but if you opening one makes it more likely to land on `master` sooner, let's do that :)